dev/io.openliberty.security.oidcclientcore.internal/resources/io/openliberty/security/oidcclientcore/internal/resources/OidcClientCoreMessages.nlsprops

```diff
@@ -33,9 +33,9 @@ DISCOVERY_EXCEPTION=CWWKS2403E: The {0} OpenID Connect client encountered the fo
 DISCOVERY_EXCEPTION.explanation=The OpenID Connect client configuration might be missing information, or the client encountered an error while communicating with the OpenID Connect provider.
 DISCOVERY_EXCEPTION.useraction=See the error in the message for more information.
 
-OIDC_CLIENT_MISSING_PROVIDER_URI=CWWKS2404E: The providerURI attribute for the {0} OpenID Connect client is null or empty, and there is no metadata available for the OpenID Connect provider.
-OIDC_CLIENT_MISSING_PROVIDER_URI.explanation=The OpenID Connect client must discover metadata for the OpenID Connect provider, but the client does not have a providerURI attribute or provider metadata configured.
-OIDC_CLIENT_MISSING_PROVIDER_URI.useraction=Specify a value for the providerURI attribute in the OpenID Connect client configuration, or specify OpenID Connect provider metadata in the client configuration.
+OIDC_CLIENT_MISSING_PROVIDER_URI=CWWKS2404E: The OpenID Connect client cannot perform discovery because the providerURI attribute for the {0} OpenID Connect client is null or empty.
+OIDC_CLIENT_MISSING_PROVIDER_URI.explanation=The OpenID Connect client must discover metadata for the OpenID Connect provider, but the client does not have a providerURI attribute configured.
+OIDC_CLIENT_MISSING_PROVIDER_URI.useraction=Specify a value for the providerURI attribute in the OpenID Connect client configuration, or ensure the OpenID Connect provider metadata in the client configuration contains all of the data that is required by the OpenID Connect client.
```

Comment:
i wonder if this should be a warning instead... otherwise an error will be produced every time someone tries to authenticate when `providerUri` is not set and `OpenIdProviderMetadata.userinfoEndpoint` (optional) is not set.

Reply:
Yeah, I think that's sensible. I can change this to a warning.